toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/LocalSceneView.kt

```diff
@@ -62,7 +64,37 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 
 /**
- * A compose equivalent of the view-based [LocalSceneView].
+ * A user interface control that displays geographic content defined by a local [ArcGISScene].
+ * A local scene view is a user interface that displays layers and graphics. It controls the area of
+ * the local scene that is visible and supports user interactions such as pan and zoom. The local
+ * scene view also provides access to the underlying layer data in a local scene.
+ *
+ * To display a local scene, add this view to your app and assign a local scene to it via
+ * [LocalSceneView.scene]. This loads the scene and its content, such as a [Basemap] and a collection
+ * of operational layers, and displays this content on screen.
+ *
+ * User interactions such as pan, zoom, rotate, and identify or selection are supported in the
+ * [LocalSceneView] using touch, mouse, keyboard or pen/pencil. If required, you can override keys,
+ * clicks, and gestures to provide a specific user experience.
```

Comment:
Just noticed that this doesn't fully apply. (Will also need changing on the View-based LocalSceneView, SceneView and MapView):
```suggestion
 * User interactions such as pan, zoom, rotate, and identify or selection are supported in the
 * [LocalSceneView] using touch interaction.
```